src/wwwroot/partials/settings/billing.html

```diff
@@ -26,45 +26,40 @@ <h3 class="special">{{ 'billing_address_title' | translate }}</h3>
               <label class="special uppercase" for="address">{{'address_text' | translate}}</label>
             </div>
             <div class="flex three flex--wrap">
-              <input id="address" name="address" ng-model="vm.address" ng-class="(vm.submitted || form.address.$dirty) && form.address.$invalid ? 'error' : ''" class="input input--large" required />
-              <div class="width--full animate-error-container" ng-class="(vm.submitted || form.address.$dirty) && form.address.$invalid ? 'show' : 'hide'">
-                <span class="input--error" ng-show="(vm.submitted || form.address.$dirty) && form.address.$error.required"><i class="arrow--up"></i>{{'validation_error_required' | translate}}</span>
-              </div>
+              <input id="address" name="address" ng-model="vm.address" class="input input--large" validation-errors-fluid required />              
             </div>
           </div>
           <div class="item flex">
               <div class="flex one flex--vertically-start">
                 <label class="special uppercase" for="city">{{'city_label' | translate}}</label>
               </div>
               <div class="flex three flex--vertically-start city--container">
-                <input id="city" name="city" ng-model="vm.city" ng-class="(vm.submitted || form.city.$dirty) && form.city.$invalid ? 'error' : ''" class="input input--large" required />
-                <span class="input--error" ng-show="(vm.submitted || form.city.$dirty) && form.city.$error.required"><i class="arrow--up"></i>{{'validation_error_required' | translate}}</span>
-                <div class="flex flex--vertically-start zip--container">
-                  <label class="special uppercase" for="zCode">{{'zip_code_label' | translate}}</label>
-                  <input id="zCode" name="zCode" ng-model="vm.zCode" ng-class="(vm.submitted || form.zCode.$dirty) && form.zCode.$invalid ? 'error' : ''" class="input input--small width--half" required />
-                  <span class="input--error" ng-show="(vm.submitted || form.zCode.$dirty) && form.zCode.$error.required"><i class="arrow--up"></i>{{'validation_error_required' | translate}}</span>
+                <div class="flex one flex--vertically-start flex--wrap">
+                  <input id="city" name="city" ng-model="vm.city"class="input input--large" validation-errors-fluid required />
+                </div>
+                <div class="flex one flex--vertically-start zip--container">
+                  <div class="flex one flex--vertically-start flex--wrap label">
+                    <label class="special uppercase" for="zCode">{{'zip_code_label' | translate}}</label>
+                  </div>
+                  <div class="flex three flex--vertically-start flex--wrap">
+                    <input id="zCode" name="zCode" ng-model="vm.zCode" validation-errors-fluid class="input input--small width--half" required />
+                  </div>
```

Comment:
The old validation added manually doesn't had a container, and now it's going to have one, that's why i have to change the sctructure of the html.